fenix/app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,17 @@
        Common myths about private browsing
     </string>
 
+    <!-- True Private Browsing Mode -->
+    <!-- Title for description card on private homescreen in True Private Browsing Mode. -->
+    <string name="felt_privacy_desc_card_title">Leave no traces on this device.</string>
+    <!-- Explanation for private browsing displayed to users on home view when they first enable
+        private mode in our new Total Private Browsing mode.
+        The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="felt_privacy_desc_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. Who might be able to see my activity?</string>
+    <!-- Clickable portion of the explanation for private browsing that links the user to our
+        about privacy page. -->
+    <string name="felt_privacy_desc_card_subtitle_link_portion">Who might be able to see my activity?</string>
```

Comment:
```suggestion
    <string name="felt_privacy_desc_card_subtitle_link_text">Who might be able to see my activity?</string>
```
Let's reword the localization comment closer to [juno_onboarding_default_browser_description_link_text](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#302). The important part is to say this is part of another string.